Contributing.md

```diff
@@ -0,0 +1,70 @@
+# To contribute to this repository, follow the steps below:
+
+## Step 1: Fork this repository
+
+●Click the "Fork" button at the top right corner of this repository's page.
+
+![Alt text](../../Users/Lenovo/Documents/fork.png)
+
+## Step 2:Clone the forked repository
+
+●Go to your GitHub account and open the forked repository.
+●Click the "Code" button and then the copy icon to copy the repository's URL.
+
+![Alt text](../../Users/Lenovo/Documents/code.png)
```

Comment:
It appears that there is an issue with the image URL provided in the code snippet. The path `"../../Users/Lenovo/Documents/fork.png"` suggests a local file path on your personal computer, which cannot be accessed or displayed on the internet.

Reply:
Should I add the images to a file in the project?

Reply:
yes you can upload the image in GitHub itself here